xrpVendordep/src/main/native/cpp/xrp/XRPServo.cpp

```diff
@@ -44,28 +47,28 @@ XRPServo::XRPServo(int deviceNum) {
   }
 }
 
-void XRPServo::SetAngle(double angleDegrees) {
-  if (angleDegrees < 0.0) {
-    angleDegrees = 0.0;
+void XRPServo::SetAngle(units::radian_t angleRadians) {
+  if (angleRadians < 0.0_rad) {
```

Comment:
Just do `angle = units::math::clamp(angle, 0_deg, 180_deg);`. It's in the `<units/math.h>` header.